File: multi_lang/node_templates/model_template.js
// Template: Mongoose model (raw). Placeholders substituted by NodeModelGenerator.
const mongoose = require('mongoose');

const __MODEL_NAME__Schema = new mongoose.Schema({
  name: { type: String, required: true },
  price: { type: Number, required: true }
});

module.exports = mongoose.model('__MODEL_NAME__', __MODEL_NAME__Schema);

File: multi_lang/rust_templates/model_template.rs
// Template: Diesel model (raw). Placeholders substituted by RustModelGenerator.
use diesel::prelude::*;
use serde::{Deserialize, Serialize};

#[derive(Queryable, Selectable, Serialize, Deserialize)]
#[diesel(table_name = crate::schema::__MODEL_SNAKE__)]
#[diesel(check_for_backend(diesel::pg::Pg))]
pub struct __MODEL_NAME__ {
    pub id: i32,
    pub name: String,
    pub price: f64,
}